.github/workflows/build.yml

```diff
@@ -78,6 +78,9 @@ jobs:
             bundler-cache: true
       
         - run: bin/setup
+        - if: steps.changes.outputs.ruby == 'true'
+          name: run zeitwerk:check
+          run: bin/rails zeitwerk:check
```

Comment:
Add a check so we know we're staying Zeitwerk compatible

Reply:
Makes sure classes load correctly?

Reply:
Yes, that's part of it. It also makes sure the files they're loading from match what Zeitwerk expects.